content/getting-started/events-api-best-practices/best-practices.md

```diff
@@ -0,0 +1,88 @@
+# Best practices to use the Events API with your connector
+
+<!--
+- Explication rapide de l'events api + lien vers la référence doc de l'events api + helpcenter
+- Explication rapide d'un use-case d'une extension connecté à l'events api
+  - aka, mon site ecommerce aimerait se sync avec le pim rapidement aprés des modifications sur un produit
+  -->
+
+This page presents best practices and recommendations you might need while implementing the Events API inside an API based connector.
+
+First, if you need help to setup a connector to use the Events API take a loot at the [Quick start my first webhook](/getting-started/quick-start-my-first-webhook/welcome.html) guide.
+
+## Configure your server
+
+TODO
```

Comment:
?

Reply:
I will hide it for now, before merging, and we will complete it at the end of the week with PY